src/devTools.js

```diff
@@ -95,13 +110,26 @@ class DevToolsEnhancer {
     }
   }
 
+  importPayloadFrom = (store, state, instance) => {
+    try {
+      const nextLiftedState = importState(state, instance);
```

Comment:
We could manually construct the `instance` here from `this.instanceId`, `this.instanceName`, etc.